### PR TITLE
p3d to sylphase for indyav urdf

### PR DESCRIPTION
**Pull request by alexperez33**
added sylphase and fixed_link_persistent xacros to xacro folder as they are required to add sylphase to indyav urdf. deleted p3d xacro instance and include, and added the same sylphase instance as from the navigator urdf

### CODE REVIEW

File: IndyAV/simulation/indyav_gazebo/xacro/fixed_link_persistent.xacro [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```
Comment (MarshallRawson) [inline IndyAV/simulation/indyav_gazebo/xacro/fixed_link_persistent.xacro:1]: This file shouldnt be included as it is a copy of the one in Navigator.

File: IndyAV/simulation/indyav_gazebo/xacro/sylphase.xacro [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```
Comment (MarshallRawson) [inline IndyAV/simulation/indyav_gazebo/xacro/sylphase.xacro:1]: This file is a copy of the one in Navigator

**Review by MarshallRawson — CHANGES_REQUESTED** [review]
Leave a comment describing how to demonstrate the changes made

**Comment by alexperez33** [comment]
roslaunch indyav_launch gazebo.launch
rostopic list //shows the topic "ins_odom"
rostopic echo /ins_odom //shows the topics info successfully

**Comment by MarshallRawson** [comment]
> roslaunch indyav_launch gazebo.launch
> rostopic list //shows the topic "ins_odom"
> rostopic echo /ins_odom //shows the topics info successfully

pro tip:
git hub comments are in markdown!

`roslaunch indyav_launch gazebo.launch`
`rostopic list #shows the topic ins_odom`
`rostopic echo #ins_odom //shows the topics info successfully`

**Comment by MarshallRawson** [comment]
> roslaunch indyav_launch gazebo.launch
> rostopic list //shows the topic "ins_odom"
> rostopic echo /ins_odom //shows the topics info successfully

Great! Also, could you write a ros test and put it under the `IndyAV/localization/test` directory. Also, go and update the documentation under `docs/indyav/software/simulation/gazebo/gazebo.dot` and change the p3d to syphaze Also, update the `docs/indyav/software/localization.rst` to say that we are currently using the Sylphase INS

**Comment by alexperez33** [comment]
To demo the change,
`roslaunch indyav_launch gazebo.launch`
Then in another terminal
`cd ~/mil_ws/src/mil/IndyAV/localization/indyav_localization/test`
`./test_sylphase.py`

File: IndyAV/command/indyav_launch/launch/odometry.launch [hunk 3]
```diff
@@ -1,9 +1,17 @@
 <?xml version="1.0"?>
 <launch>
+
   <arg name="environment" default="real"/>
-  <!-- TODO: replace this with a proper odom estimator -->
-  <node if="$(eval environment == 'gazebo')" 
+  <!-- This file is no longer being run -->
```
Comment (MarshallRawson) [inline IndyAV/command/indyav_launch/launch/odometry.launch:5]: delete this commented out node from this launch file

File: IndyAV/localization/indyav_localization/test/test_sylphase.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    sylphase_pub = False
+    sylphase_hz_rate = 0
+    high_hz = False
+
+    def callback(self, data):
+        self.sylphase_hz_rate += 1
+
+    
+    def test_ins_odom_frequency(self):
+        sleep(8) #allows ample time for sim to start up
```
Comment (MarshallRawson) [inline IndyAV/localization/indyav_localization/test/test_sylphase.py:18]: replace with `rospy.wait_for_service("/gazebo/spawn_urdf_model")`

File: IndyAV/localization/indyav_localization/test/test_sylphase.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        rospy.init_node('test_sylphase')
```
Comment (MarshallRawson) [inline IndyAV/localization/indyav_localization/test/test_sylphase.py:19]: remove this line as this tis taken care of on line 40

File: IndyAV/simulation/indyav_gazebo/urdf/indyav_car.urdf.xacro [hunk 6]
```diff
@@ -14,10 +15,11 @@
     </collision>
   </link>
 
-  <!-- TODO: replace this with a sylphase img gps -->
-  <xacro:property name="namespace" value="indyav_car"/>
-  <xacro:property name="sensor_namespace" value="sensors"/>
-  <xacro:wamv_p3d/>
+  <!-- <xacro:property name="namespace" value="indyav_car"/> -->
```
Comment (MarshallRawson) [inline IndyAV/simulation/indyav_gazebo/urdf/indyav_car.urdf.xacro:18]: delete these two commented out properties

**Comment by MarshallRawson** [comment]
the test is failing because the ros node in the test is being re-initialized on like 19

**Review by MarshallRawson — CHANGES_REQUESTED** [review]

**Review by MarshallRawson — APPROVED** [review]

**Pull request merged**